frontend/src/app/ui/navbar/navbar.component.ts

```diff
@@ -35,6 +38,9 @@ export class NavbarComponent implements OnInit {
     }
   }
 
+  /**
+   * search plan
```

Comment:
Wird ja keine Suche ausgeführt, sondern nur auf die Seite weitergeleitet

Redirect to search overview page